src/cli/cli_mac_filter.cpp

```diff
@@ -156,15 +188,65 @@ template <> otError MacFilter::Process<Cmd("addr")>(Arg aArgs[])
             SuccessOrExit(error = otLinkFilterAddRssIn(GetInstancePtr(), &extAddr, rss));
         }
     }
+    /**
+     * @cli macfilter addr remove
+     * @code
+     * macfilter addr remove 0f6127e33af6b402
+     * Done
+     * @endcode
+     * @cparam macfilter addr remove @ca{extaddr}
+     * @par api_copy
+     * #otLinkFilterRemoveAddress
+     */
     else if (aArgs[0] == "remove")
     {
         SuccessOrExit(error = aArgs[1].ParseAsHexString(extAddr.m8));
         otLinkFilterRemoveAddress(GetInstancePtr(), &extAddr);
     }
+    /**
+     * @cli macfilter addr clear
+     * @code
+     * macfilter addr clear
+     * Done
+     * @endcode
+     * @par api_copy
+     * #otLinkFilterClearAddresses
+     */
     else if (aArgs[0] == "clear")
     {
         otLinkFilterClearAddresses(GetInstancePtr());
     }
+    /**
+     * @cli macfilter addr disable
```

Comment:
I wonder if we should move these before each constant in kModeCommands below.  @abtink thoughts?

Reply:
Yes. Good idea. Make sense to me to move them next to related constants.